### PR TITLE
chore(release): v10.26.0

### PR DESCRIPTION
**Pull request by andreancardona**
Release PR for v10.26.0

### CODE REVIEW

File: config/storybook-preset-carbon/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Change Log
```
Comment (joshblack) [inline config/storybook-preset-carbon/CHANGELOG.md]: Seems like it might have committed some changelogs that we'll need to remove 😅

**Review by joshblack — CHANGES_REQUESTED** [review]

**Review by aledavila — APPROVED** [review]

File: packages/pictograms/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@carbon/pictograms",
   "description": "Pictograms for digital and software products using the Carbon Design System",
-  "version": "11.1.0-rc.1",
+  "version": "12.0.0",
```
Comment (joshblack) [inline packages/pictograms/package.json]:
```suggestion
  "version": "11.1.0",
```

File: packages/pictograms-react/package.json [hunk 3]
```diff
@@ -37,13 +37,13 @@
...
-    "@carbon/icon-helpers": "^10.11.0-rc.0",
+    "@carbon/icon-helpers": "^10.11.0",
     "@carbon/telemetry": "0.0.0-alpha.6",
     "prop-types": "^15.7.2"
   },
   "devDependencies": {
-    "@carbon/icon-build-helpers": "^0.13.0-rc.0",
-    "@carbon/pictograms": "^11.1.0-rc.1"
+    "@carbon/icon-build-helpers": "^0.13.0",
+    "@carbon/pictograms": "^12.0.0"
```
Comment (joshblack) [inline packages/pictograms-react/package.json]:
```suggestion
    "@carbon/pictograms": "^11.1.0"
```

File: packages/react/package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "carbon-components-react",
   "description": "The Carbon Design System is IBM’s open-source design system for products and experiences.",
-  "version": "7.26.0-rc.1",
+  "version": "8.0.0",
```
Comment (joshblack) [inline packages/react/package.json]:
```suggestion
  "version": "7.26.0",
```

File: packages/components/package.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "carbon-components",
   "description": "The Carbon Design System is IBM’s open-source design system for products and experiences.",
-  "version": "10.26.0-rc.1",
+  "version": "11.0.0",
```
Comment (joshblack) [inline packages/components/package.json]:
```suggestion
  "version": "10.26.0",
```

**Review by joshblack — CHANGES_REQUESTED** [review]
Seems like a couple breaking changes might have snuck in there

File: packages/cli/src/workspace.js [hunk 6]
```diff
@@ -56,8 +56,7 @@ async function getPackages() {
   return JSON.parse(
     // Clean-up output by stripping out `yarn` information related to the
     // command and how long it took to run
-    lernaListOutput.split('\n').slice(2, -1).join('\n')
-  ).filter((pkg) => !pkg.private);
+    lernaListOutput.split('\n').slice(2, -1).join('\n').filter((pkg) => !pkg.private);
```
Comment (joshblack) [inline packages/cli/src/workspace.js]: Seems like a prettier change that might not need to be included

**Review by joshblack — APPROVED** [review]

**Review by joshblack — APPROVED** [review]

**Pull request merged**